ocean/region/Bass_Strait_North/region.geojson

```diff
@@ -3,9 +3,9 @@
 	[
 		{"type": "Feature",
 		 "properties": {
-			"name": "Bass Strait",
+			"name": "Bass Strait North",
 			"component": "ocean",
-			"tags": "",
+			"tags": "Bass_Strait;Indian_Basin",
```

Comment:
@milenaveneziani, the Bass Strait looks okay here.  Maybe you need to do a `git fetch xylar/geometry_features` and then a `git reset --hard xylar/geometry_features/tag_ocean_basins`?